cms/browse/views.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from django.shortcuts import render
+from wagtail.core.models import Page
+
+from cms.core.models import ExtendedMainMenuItem
+
+logger = logging.getLogger("general")
+
+
+def browse(request, programme, branch):
+    # we could get current site, query main menu for matching items and then
+    # load the items but we _know_ that ExtendedMenuItems will haveour list
+    # in it. 4+ queries -> 1 query
+    menu_items = ExtendedMainMenuItem.objects.all()
```

Comment:
This is true at the moment, but it's possible that we could add a second menu at some point which also used `ExtendedMainMenuItem`s, and this would explode in unexpected ways.

Is there a simple way to query for only the things on the menu?